pages/k8s/metallb.md

```diff
@@ -42,13 +42,30 @@ explanation of the required configuration</a> from the
 
 </p></div>
 
-Currently, the best way to install MetalLB on Charmed Kubernetes is with a helm chart:
+The best way to install MetalLB in layer 2 mode on Charmed Kubernetes is with 
+the MetalLB Operator. The MetalLB Operator is a charm bundle that allows the 
+deployment of both the controller and speaker components.
 
 ```bash
-helm install --name metallb stable/metallb
+juju add-model metallb-system
```

Comment:
It's possible that the reader will not have created a k8s cloud yet. We should include instructions for that here. See the [Multus docs](https://ubuntu.com/kubernetes/docs/cni-multus) for an example.